packages/manager/core/generator/app/index.js

```diff
@@ -201,6 +202,33 @@ export default (plop) => {
         },
         validate: (input) => input.length > 0,
       },
+      {
+        type: 'list',
+        name: 'level2',
+        message: 'What is the level2 of the app ? (tracking)',
+        choices: Object.keys(LEVEL2).map((element) => ({
+          name: `${element} - ${LEVEL2[element]}`,
+          value: element,
+        })),
+      },
```

Comment:
As I already mentioned in my previous comment, LEVEL2 can be different based on the region. So, I think we should accept it for each region.

Reply:
👍